packages/tpetra/core/src/Tpetra_Details_makeOptimizedColMap.hpp

```diff
@@ -88,8 +88,7 @@ namespace Details {
     makeOptColMap (std::ostream& errStream,
                    bool& lclErr,
                    const map_type& domMap,
-                   const map_type& colMap,
-                   const import_type* /* oldImport */)
+                   const map_type& colMap)
```

Comment:
It's actually still worthwhile to have an option to pass in the old Import.  The point is that we could use the information in the old Import object, if provided, to construct the optimized column Map with less communication.  (For example, the old Import knows what GIDs in the input column Map are local vs. remote, and it knows the remote GIDs' owning PIDs.)  We don't implement that optimization now, but it wouldn't be too hard to add later, and the current interface permits that.